### PR TITLE
Fix(web-twig): Fix pass custom classnames to the `Spinner` component

### PR DESCRIPTION
**Pull request by dlouhak**

### CODE REVIEW

File: packages/web-twig/src/Resources/components/Spinner/Spinner.twig [hunk 1]
```diff
@@ -11,8 +11,8 @@
 {%- set _classNames = [ _rootClassName, _rootAnimationClassName, _styleProps.className ] -%}
 {%- set _mainPropsWithoutColor = props | filter((value, prop) => prop is not same as('color')) -%}
 
-{% embed "@spirit/icon.twig" with { props: {
+{% embed "@spirit/icon.twig" with { props: _mainPropsWithoutColor | merge({
```
Comment (dlouhak) [inline packages/web-twig/src/Resources/components/Spinner/Spinner.twig:14]: Credit to @crishpeen.

**Comment by crishpeen** [comment]
No description provided? Can you please explain how this happened so we can prevent it in the future?

File: packages/web-twig/src/Resources/components/Spinner/Spinner.twig [hunk 2]
```diff
@@ -11,8 +11,8 @@
...
     name: 'spinner',
     UNSAFE_className: _classNames | join(' ') | trim,
     UNSAFE_style: _styleProps.style,
-} | merge(_mainPropsWithoutColor) } %}{% endembed %}
+})} %}{% endembed %}
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/components/Spinner/Spinner.twig:18]:
```suggestion
}) } %}{% endembed %}
```

**Review by crishpeen — APPROVED** [review]

**Comment by dlouhak** [comment]
> No description provided? Can you please explain how this happened so we can prevent it in the future?

Git blame will answer your questions. 🙃 
Thorough code review (✅) and complex snapshot tests (✅).

**Review by literat — APPROVED** [review]

**Pull request merged**